### DISCUSSION TITLE
Need help, i dont know where else to ask, want to know how to upload multiple folders to the same repo

### DISCUSSION THREAD

**Discussion by MatGS123** [community/community#143126] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi, i wanted help with uploading multiple folders and files to the same repo. I have a main folder, inside there are the front and back folders. So, when i try to upload, it only lets me upload the back folder, a single folder, how can i upload many folders? is it impossible? they have thousands of files inside, and using the create file with the slash method not feasible. Anyone knows what to do? I already checked the git ignote and file size limit, neither are a problem.

**Comment by TzivyaFireman** [comment, marked as answer]
Hey,
If you want to upload multiple folders to the same repo, like your "front" and "back" folders within a main directory, here’s what you can try:

Stage the Main Folder: In the command line, navigate to your main folder that contains both subfolders and run git init if it's a new repo.
Then use git add . to stage all files and folders within it.

Commit and Push Everything: Once all files are staged, commit them with git commit -m "Initial commit" and push with git push origin main (or replace "main" with the relevant branch name).

If git add . only stages one folder, check if any gitignore file is in effect, even indirectly (sometimes, IDEs or previous settings can interfere). This approach will ensure both "front" and "back" folders are pushed together, as long as nothing is excluded or restricted.

**Reply by MatGS123** [reply]
Thanks, i was able to solve it
